homework/remove-vowels/vowel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "vowel.hpp"
+
+void removeVowels(std::vector<std::string> & vec_string)
+{
+    const std::string vowel_string("aeiouyAEIOUY");
+
+    for (auto & el : vec_string)
+    {
+        for (auto & el_char : vowel_string)
```

Comment:
It would be better not to use reference here. `vowel_string` contains values of type `char` which is of size 1 byte. Reference is of size 4 bytes, so this reference is not optimizing anything here.